input/P15tSZ.yml

```diff
@@ -0,0 +1,12 @@
+mapper_class: MapperP15tSZ
+map_name: P15tSZ
+file_map: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-milca-ymaps_2048_R2.00.fits
+file_noise: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-milca-stddev_2048_R2.00.fits
+file_mask: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-masks_2048_R2.01.fits
+gp_mask_mode: '0.5'
+ps_mask_mode: ['default']
+beam_info: [{'type': 'Gaussian', 'FWHM_arcmin': 10.0}]
+coords: G
+custom_auto: True
+mask_name: mask_P15tSZ
+path_rerun: '/mnt/extraspace/damonge/Datasets/P15tSZ/xcell_runs'
```

Comment:
Don't think this path exists

Reply:
The folder doesn't exist but I have now pointed it out to a more apropiate path. Would we need to create the rerun folder in advance or would it be created after the run if it doesn't exist yet?

Reply:
This happens within the code, but if the preceding folder doesn't exist, an error will be thrown